charm/ksql/config.yaml

```diff
@@ -7,15 +7,11 @@ options:
     type: string
     default: development
     description: The KSQL cluster id that the KSQL server will be assigned to.
-  ksql-listener-port:
-    type: int
-    default: 8088
-    description: The port where the REST server will listen on
-  ksql-listener-protocol:
-    type: string
-    default: http
-    description: The protocol that the REST server will listenn on
   use-ssl:
     type: boolean
     default: true
-    description: Should KSQL connect to Kafka via SSL orr plaintext
\ No newline at end of file
+    description: Should KSQL connect to Kafka via SSL orr plaintext
+  ksql-should-listen-private:
```

Comment:
Charm should only support listening on the private address and possibly also localhost (to allow ksql batch scripts to run locally). There's really not much of a use case for putting KSQL on the public internet. For development purposes, one can use LXD in which case private IP == public IP.